global-news-app/src/components/Search/Search/Search.styles.tsx

```diff
@@ -0,0 +1,16 @@
+import styled from "styled-components";
+
+export const SearchContainer = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background: ${({ theme }) => theme.colors.white};
+  border-radius: 10px;
+  height: 50px;
+  max-width: 423px;
+  padding-left:17.75px;
```

Comment:
let's not use decimals for spacing

make sure that you files are formatted the same, this will not allow to avoid unnecessary changes in next prs